src/helpers/file-helper.js

```diff
@@ -14,22 +14,61 @@ export const getExtension = (file) => {
  */
 export const isValidFile = (file, whitelist) => {
   const extension = getExtension(file);
-  return whitelist.indexOf(extension) >= 0;
+  return whitelist.includes(extension);
```

Comment:
I wonder if we should `.toLowerCase()` the extension here? Will it matter if the extension is `JPEG` for example?

Reply:
You are right, we should normalise the string

Reply:
done